.github/workflows/ci.yml

```diff
@@ -33,11 +33,13 @@ jobs:
         fetch-depth: 0
     - uses: nrwl/nx-set-shas@177b48373c6dc583ce0d9257ffb484bdd232fedf
       id: last_successful_commit_push
+      if: ${{ inputs.plugins != '' }}
       with:
         main-branch-name: ${{ github.ref_name }}
         workflow-id: 'ci.yml'
     - name: Get changed files
       id: changed-files
+      if: ${{ inputs.plugins != '' }}
```

Comment:
Tried kicking off a manual build to rebuild an individual plugin and got https://github.com/bufbuild/plugins/actions/runs/4557846013/jobs/8040014496. This should skip all those steps if we've set the workflow variable already.